<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoAtlasProfiles</class>
 <widget class="QWidget" name="DemoAtlasProfiles">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>418</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
     <item>
      <widget class="PwGisWidget" name="pwgiswidget" native="true">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>170</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="mapLayout"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="atlasLayout">
           <item>
            <widget class="QPushButton" name="btnOpenAtlas">
             <property name="text">
              <string>Open Atlas</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCloseAtlas">
             <property name="text">
              <string>Close Atlas</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="profileLayout">
           <item>
            <widget class="QPushButton" name="btnRegisterProfile">
             <property name="text">
              <string>Register profile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnUnregisterProfile">
             <property name="text">
              <string>Unregister profile</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PwGisWidget</class>
   <extends>QWidget</extends>
   <header location="global">PwGis/pwgiswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
